### ISSUE TITLE
Always work with local[*], does not connect to cluster even with MASTER specified

### ISSUE THREAD

**Issue by eranation** [NFLabs/zeppelin#325]
I'm using a build for spark  1.1 + hadoop 2.4 (same as my cluster)
I tried setting conf/zeppelin-env.sh MASTER property and didn't work.
looking at the sources, I also tried adding spark.conf to conf/zeppelin-site.xml, but it didn't work 

(by the way the sys property didn't show using sys.props using the site.xml method, only after I added it with ZEPPELIN_JAVA_OPTS for some reason... but this is another issue perhaps)

I have a running spark cluster on "mysparkmaster" (I double checked with a spark console) 

Bottom line, I ran this from a notebook (I interleaved the results for clarity):

sys.props("spark.master")
res5: String = spark://mysparkmaster:7077

sys.env("MASTER")
res6: String = spark://mysparkmaster:7077

sc.master
res7: String = local[*]

What am I missing?

**Comment by dszeto** [comment]
Same issue here.

**Comment by dszeto** [comment]
I figured this out. You have to create a new interpreter with your Spark settings in the Interpreter tab (next to Notebook), and set your notebook to use this new interpreter.

**Comment by eranation** [comment]
Thanks dezeto, you are right. But for some reason this doesn't work either (in a different way...)

Now I get this error message 

    org.apache.spark.SparkException: Job aborted due to stage failure: All masters are unresponsive!   Giving up.

spark-console is connecting to that master without a problem.

**Comment by swkimme** [comment]
@eranation 
Hi Eran.

Did you start spark-daemon using spark/sbin/start-all.sh ??
Spark-shell tha t used to test the connection with master has run on same machine with Zeppelin?

If not, I recommend to check your network settings.

And I guess we need some document or tutorial on adding an interpreter toward remote cluster..
(Setting MASTER in zeppelin-env.sh is an old way, editing master prop. in interpreter setting is a new way.)
